index.html

```diff
@@ -1986,22 +1854,21 @@ <h3>The <dfn>NFCReaderOptions</dfn> dictionary</h3>
           <li>
             If there is no underlying <a>NFC Adapter</a>, or if a connection cannot
             be established, then reject <var>p</var> with a
-            <code>"<a>NotReadableError</a>"</code> <code><a>DOMException</a></code>
+            "{{NotReadableError}}" {{DOMException}}
```

Comment:
Same for the rest... landed support for this today!:

```suggestion
            {{"NotReadableError"}} {{DOMException}}
```